Tests/test_AlignIO_FastaIO.py

```diff
@@ -309,7 +309,7 @@ def test_example(self):
         alignments = list(FastaM10Iterator(StringIO(simple_example)))
         assert len(alignments) == 4, len(alignments)
         assert len(alignments[0]) == 2
-        self.assertEqual(% (len(a), a.get_alignment_length()))
+        self.assertEqual(%(len(a), a.get_alignment_length()))
```

Comment:
What is this percent sign doing? Surely you just want to compare two integers with the syntax ``self.assertEqual(value_one, value_two)`` thus:

``` python
self.assertEqual(len(a), a.get_alignment_length())
```